environment.yml

```diff
@@ -8,13 +8,18 @@ dependencies:
   - pandas
   - matplotlib
   - jupyter
+  - pytorch
+  - torchvision
+  - opencv
 
   # dev
   - black
   - flake8
 
   - pip
   - pip:
+    - imageio
+    - scipy
```

Comment:
I would not install scipy with pip when numpy and pandas are installed with conda

can you send a new PR to try this?

thx @alibell @frcaud